### PR TITLE
Double count session attendance fix

### PR DESCRIPTION
**Pull request by bam95899**

### CODE REVIEW

File: sciencelabs/db_repository/session_functions.py [hunk 1]
```diff
@@ -76,6 +76,13 @@ def get_session_students(self, session_id):
             .filter(StudentSession_Table.studentId == User_Table.id)\
             .all()
 
+    def get_number_of_sessions(self, session_id):
```
Comment (c-schwarze) [inline sciencelabs/db_repository/session_functions.py]: should this be, `get_number_of_student_sessions`?

File: sciencelabs/reports/__init__.py [hunk 2]
```diff
@@ -73,7 +74,7 @@ def export_student_csv(self):
         my_list = [['Last', 'First', 'Email', 'Attendance']]
 
         for student, attendance in self.user.get_student_info(session['SELECTED-SEMESTER']):
-            my_list.append([student.lastName, student.firstName, student.email, attendance])
+            my_list.append([student.lastName, student.firstName, student.email, len(self.user.get_unique_sessions_attended(student.id, session['SELECTED-SEMESTER']))])
```
Comment (c-schwarze) [inline sciencelabs/reports/__init__.py]: this line just made me think of it. I assume you have been copying the current export of reports to base these off of?

File: sciencelabs/db_repository/session_functions.py (hunk 1 above)
Comment (bam95899) [reply sciencelabs/db_repository/session_functions.py]: changed

File: sciencelabs/reports/__init__.py (hunk 2 above)
Comment (bam95899) [reply sciencelabs/reports/__init__.py]: Yup, I have been copying the format/data from the current Math Lab site.

**Pull request merged**